### ISSUE TITLE
Problems with incremental packager on Kaniko

### ISSUE THREAD

**Issue by nicolaferraro** [apache/camel-k#998]
I know there was already an issue for this, but I think they are two different problems.

I did this:
1. Create a knative integration targeting service a and make it run
2. Restart the cluster (minikube)
3. Edit the integration to target service b, since there's another bug that make the two kits different, another build is started, starting from the result of the first one

The Kaniko pod throws an error:
```
INFO[0000] Downloading base image 10.99.251.236/default/camel-k-kit-bmeffspsmgr6cu5thqkg:77013 
2019/10/09 14:48:48 No matching credentials were found, falling back on anonymous
INFO[0000] Error while retrieving image from cache: MANIFEST_UNKNOWN: "manifest unknown"
```

**Comment by nicolaferraro** [comment]
Not so obvious to replicate.. It's now working for me

**Comment by astefanutti** [comment]
To be assessed after #1121 is merged.

**Comment by nicolaferraro** [comment]
Duplicate of #1080
